src/lib/components/Logo.tsx

```diff
@@ -0,0 +1,58 @@
+import LogoRA from "@iobroker/adapter-react-v5/Components/Logo";
+import React from "react";
+import { useDialogs } from "../hooks/useDialogs";
+import { useGlobals } from "../hooks/useGlobals";
+import { useIoBrokerObject } from "../hooks/useIoBrokerObject";
+
+interface LogoProps {
+	classes?: { buttons?: string; logo?: string };
+}
+
+export const Logo: React.FC<LogoProps> = (props): JSX.Element => {
+	const { instance, namespace } = useGlobals();
+	const { showNotification } = useDialogs();
+	const [myObject, setObject] = useIoBrokerObject(
```

Comment:
The 2nd array item is `extendObject`, not `setObject`. What exactly is this supposed to do?

Reply:
I had tried it with extendObject, but that did not have the desired effect with the `extendObject` a deep merge is made, with the `handleLoadConfig`, where setObject is used, the old config is reloaded, but it should not be merged, but overwritten or am I wrong?